Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/")
+def home():
+  return "You are in the right place"
+
+@app.route("/<opp>/<num1>/<num2>")
+def do_math(opp, num1, num2):
+  if opp == "add":
+    return str(int(num1) + int(num2))
+  elif opp == "subtract":
+    return str(int(num1) - int(num2))
+  elif opp == "multiply":
+    return str(int(num1) * int(num2))
+  elif opp == "divide":
+    return str(float(num1) / int(num2))
+	
+@app.route("/math/<opp>/<num1>/<num2>")
+def do_math2(opp, num1, num2):
+  if opp == "add":
+    return str(int(num1) + int(num2))
+  elif opp == "subtract":
```

Comment:
This is good; However, another way to do this is using a dictionary, just like you can in JavaScript using objects.

```python
def add(a, b):
    return a + b
def subtract(a, b):
    return a - b
def multiply(a, b):
    return a * b
def divide(a / b):
    return a / b

operations = {
    "add": add,
    "subtract": subtract,
    "multiply": multiply,
    "divide": divide
}

# @route handler
def math(operation, num1, num2):
    if operation in operations:
        return f"{operations[operation](num1, num2)}"
```

This is a more declarative style, where you can declare your functions and data structures at the time and simplify the business logic part (i.e. the routing, in this case)